api/inventorypb/agents.proto

```diff
@@ -893,6 +917,53 @@ message ChangeRDSExporterResponse {
   RDSExporter rds_exporter = 1;
 }
 
+// Add/Change ExternalExporter
+
+message AddExternalExporterRequest {
+  // The node identifier where this instance is run.
+  string runs_on_node_id = 1 [
+    (validator.field) = {
+      string_not_empty: true
```

Comment:
Our internal docs still say that zero Nodes is possible.

Reply:
Updated. Will update other parts of documentation after implementing external exporters. External exporters have higher priority then updating documentation now.

Reply:
It is the same priority – we don't have a separate ticket for updating documentation just like we don't have separate tickets for writing unit tests. Existing page in the Confluence _must_ be in sync with code.

The easiest way to ensure that is to remove everything from the Confluence that can be seen in proto files. For example, a list of non-id fields could be removed. But Confluence documentation _must_ be updated to reflect the changes in relations between types: that Node is required even for external exporters, that we have external services, etc.

Reply:
Updated documentation related to external exporters